wallet/src/de/schildbach/wallet/ui/WalletTransactionsFragment.java

```diff
@@ -147,6 +148,15 @@ public void onChange(final boolean selfChange) {
         }
     };
 
+    private InstantSendEventListener instantSendEventListener = new InstantSendEventListener() {
```

Comment:
This listener is redundant.  

The wallet has a listener for transaction confidence changes and that triggers this listener at line 563:
```
    private final ThrottlingWalletChangeListener transactionChangeListener = new ThrottlingWalletChangeListener(
            THROTTLE_MS) {
        @Override
        public void onThrottledWalletChanged() {
            adapter.notifyDataSetChanged();
        }
    };
```

Reply:
I removed the InstantSendEventListener from DashJ: [4ccd1bd](https://github.com/dashevo/dashj/commit/4ccd1bd0342c7c24dbd956124bc91fe7cab2e064)
and from the wallet: [0bedf73](https://github.com/HashEngineering/dash-wallet/pull/156/commits/0bedf7318149afb7a7d49210540f7039abb6e914#diff-2be9434186730f60270a0e6862aa8f3bL41)
The `transactionChangeListener` is registred for Transaction Confidence Events and after applying
your fix of transaction confidence changes listener the InstantSendEventListener will be needless.

Also used `IXType.IX_LOCK_FAILED` to verify the status of simple transactions: [8576794](https://github.com/HashEngineering/dash-wallet/commit/8576794d88c246fb08843bdc461242384e4c08d2)